src/Widgets/WallpaperContainer.vala

```diff
@@ -91,12 +91,15 @@ public class PantheonShell.WallpaperContainer : Gtk.FlowBoxChild {
         card_box.add (image);
         card_box.margin = 9;
 
-        var check = new Gtk.Image.from_resource ("/io/elementary/switchboard/plug/pantheon-shell/check.svg") {
+        var check_provider = new Gtk.CssProvider ();
+        check_provider.load_from_resource ("/io/elementary/switchboard/plug/pantheon-shell/Check.css");
+
+        var check = new Gtk.RadioButton (null) {
```

Comment:
Gtk.RadioButton is deprecated in Gtk4 I think.  Is there a solution that will work in Gtk4 as well?